src/main/java/de/maxhenkel/audioplayer/command/PlayCommands.java

```diff
@@ -41,4 +41,19 @@ public void play(CommandContext<CommandSourceStack> context, @Name("sound") UUID
         context.getSource().sendSuccess(() -> Component.literal("Successfully played %s".formatted(sound)), false);
     }
 
+    @RequiresPermission("audioplayer.play_command")
+    @Command("stop")
+    private static int stop(CommandContext<CommandSourceStack> context, @Name("sound") UUID sound) {
```

Comment:
The command currently just stops ALL sounds with that ID. This does not seem to be intentional. It should only stop sounds that were played with the play command. I'd suggest a boolean field marking the sound as a sound that was played with the command and filtering by that.